src/ape_test/accounts.py

```diff
@@ -126,6 +129,29 @@ def public_key(self) -> "HexBytes":
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__}_{self.index} {self.address_str}>"
 
+    def sign_authorization(
+        self,
+        address: "AddressType",
+        chain_id: Optional[int] = None,
```

Comment:
so you can pass `0` here for all chains? **idea**: also support `"*"` at the top-level because it is clearer to mean "all chains" imo

Reply:
ac873ef7a